src/esm_parser/esm_parser.py

```diff
@@ -87,6 +87,7 @@
 
 # functions reading in dict from file
 from .yaml_to_dict import *
+from .provenance import *
```

Comment:
Just a style comment, you can leave this as is: I don't particularly like wildcard imports, it makes it more difficult to see where the code is coming from.